src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -867,6 +868,11 @@ internal void ReleaseAllCores()
             }
         }
 
+        /// <summary>
+        /// Returns the minimum message importance not guaranteed to be ignored by registered loggers.
```

Comment:
nit: can use `<inheritdoc/>` to avoid copy pasting the supertype's doc

Reply:
Thank you, I didn't know this tag existed.